jdaviz/configs/imviz/plugins/tools.py

```diff
@@ -30,7 +30,7 @@ def activate(self):
 @viewer_tool
 class BqplotMatchWCS(BqplotPanZoomMode):
 
-    icon = os.path.join(ICON_DIR, 'pan_wcs.png')
+    icon = os.path.join(ICON_DIR, 'pan_wcs.svg')
```

Comment:
You can remove the PNG file from the repo too

Reply:
See #679